specification/eventgrid/data-plane/Microsoft.DataBox/stable/2018-01-01/DataBox.json

```diff
@@ -0,0 +1,74 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Data Box events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Data Box events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "DataBoxCopyStartedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.DataBox.CopyStarted event.",
```

Comment:
@JoshLove-msft  These names like "CopyStarted" are already used in our service and our customers are well versed with these, hence we are reusing them.